web-console/4-patterns/notifications.md

```diff
@@ -0,0 +1,68 @@
+# Notification Drawer
+
+## Open Questions
+- Can we bring System Alerts (Online Only) into the notification drawer?
+  - Note: Designs below assume yes.
+- Should we have an additional section for Homepage Notifications/ general catalog alerts?
+  Note: Designs below assume no.
+- Do we want to try to keep events around longer than the current 2 hour timeframe?
+- Can we get date/time info for events?
+  - Note: Designs below assume yes.
+
+## Design
+
+![Notifications](img/Catalog_Drawer-1.png)
+
+**NOTE:** Sections below in **Bold** indicate design details that stray from the PatternFly [Notification Drawer](http://www.patternfly.org/pattern-library/communication/notification-drawer/) pattern.
+
+- The bell icon is in the [Masthead](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/masthead) and provides access to the notification drawer.
+- **The fa-bell icon should always be used here. A blue badge will be shown to the top right corner of the bell to indicate when "New" notifications exist.**
+- A "New" notification is displayed with bold text because it is an unread notification. **Notifications are marked as read by clicking on the cell to unbold the text.**
+- **Users can clear notifications individually or by clicking the clear all button to remove all notifications from the currently open accordion section.**
+- There are two types of notifications that will be surfaced, System Alerts and Events.
+- An accordion style is used inside the drawer for opening and closing one section at a time.
+- If the drawer is closed and reopened, your last opened section will remain open.
+
+### System Alerts (OpenShift ONLINE Only)
```

Comment:
Maybe calling this something like "Cluster Status" or "Cluster Alerts" might be better? System seems sort of ambiguous.